support-frontend/ts-error-history.csv

```diff
@@ -59,4 +59,5 @@
 1638360727000,832,4a749cca7354efd2a096e7a7f3d53b3f43d1f951
 1638529562000,804,7cab8acd1b5c0b8aa6ba5bc50558cc2c476e19c1
 1638534645000,777,dcadc16cb35c15aba0bcd17b91133138892daf6f
-1638547419000,777,HEAD
+1638788405000,777,35d02200ed882b81b67a788f939b912ecb45e516
+1638791462000,777,HEAD
```

Comment:
I wonder if it's worth while updating the script to only add new datapoints if it's different to the previous one?